client/src/components/Navbar.js

```diff
@@ -55,6 +55,11 @@ function Navigation({ width, classes, anchorEl, handleClick, handleClose, profil
       path: "/messages",
       show: true
     },
+    {
+      title: "Find a Sitter",
+      path: "/profiles",
+		show: profile && !profile.isSitter
```

Comment:
Since we want a two sided marketplace, sitters should be able to find sitters as well - let's just make this `true`